doc/contributing.md

```diff
@@ -392,8 +392,8 @@ Guidelines for a good tutorial:
   concise examples while the tutorials are detailed and full of text.
 * SI units should be used in the example code for tutorial plots.
 
-Note that the `Figure.show()` function needs to be called for a plot to be inserted into
-the documentation.
+Note that the <code>pygmt.Figure.show</code> function needs to be called for a plot
```

Comment:
```suggestion
Note that the `pygmt.Figure.show` function needs to be called for a plot
```

Reply:
> Please note that this file is written in Markdown (strictly speaking, [MyST](https://myst-parser.readthedocs.io/en/latest/index.html)), not reStructuredText, so the syntax is different.

Yes, sure! I am sorry for this unnessecary mistake.

> Please see https://myst-parser.readthedocs.io/en/latest/index.html for the syntax of MyST, also see https://github.com/GenericMappingTools/pygmt/pull/941 for why decided to use Markdown instead of reStructuredText.

Thanks a lot for mention these links @seisman!

----------

My intention behind using `<code>` tags was to achieve the same highlighting as used for functions and methods elsewhere in the documentation:

using ``` `text` ```
![highlight_as_parameter](https://user-images.githubusercontent.com/94163266/187160678-a5974105-1e83-4ccf-8b1e-316b85df1f5d.png)
using `<code>text</code>`
![highlight_as_method](https://user-images.githubusercontent.com/94163266/187160919-57f97702-e5e2-40a6-b4e8-4b086cb6c40f.png)

Until now there are no links to the API documentation included.

Reply:
OK, I see what you want to do. Then it looks good to me.